node/include/cocaine/detail/service/node/slave/machine.hpp

```diff
@@ -186,6 +194,9 @@ class machine_t : public std::enable_shared_from_this<machine_t> {
 
     void
     dump();
+
+    ewma_type&
+    load_metric();
```

Comment:
We already have `auto machine_t::load() -> std::uint64_t` with a bit different semantics.